tools/challenge-helper-scripts/package.json

```
@@ -20,7 +20,8 @@
   "main": "utils.js",
   "scripts": {
     "test": "mocha --delay --reporter progress --bail",
-    "create-project": "tsx create-project"
+    "create-project": "tsx create-project",
+    "create-new-quiz": "tsx create-quiz"
```

```suggestion
    "create-quiz": "tsx create-quiz"
```
same pattern as above